src/SchemaReader.php

```diff
@@ -236,6 +238,28 @@ private function fillAttribute(AttributeSingle $attribute, \DOMElement $node): v
         if ($node->hasAttribute('use')) {
             $attribute->setUse($node->getAttribute('use'));
         }
+
+        $attribute->setMeta($this->loadMetaAttributesForElement($attribute, $node));
+    }
+
+    /**
+     * @return list<MetaInformation>
+     */
+    private function loadMetaAttributesForElement(SchemaItem $item, \DOMElement $node): array
+    {
+        $meta = [];
+        foreach ($node->attributes as $attr) {
+            if (null !== $attr->namespaceURI && self::XSD_NS !== $attr->namespaceURI) {
```

Comment:
isn't `null !== $attr->namespaceURI` superfluous since there is the other if condition check?


i think that it should be `if (null === $attr->namespaceURI || self::XSD_NS !== $attr->namespaceURI)`... when used as non qualified attribute... does it make sense?

Reply:
I don't think so:
It now only adds to the list when the attribute has a namespaceURI which is not XSD.
Which is what I intend it to do.